### PR TITLE
Remove extracted components

### PR DESCRIPTION
**Pull request by alexkart**
Remove components that were extracted in https://github.com/yiisoft/cache/issues/22

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,51 +1,44 @@
...
 
 env:
-  global:
-    - DEFAULT_COMPOSER_FLAGS="--prefer-dist --no-interaction --no-progress --optimize-autoloader"
+    global:
+        - DEFAULT_COMPOSER_FLAGS="--prefer-dist --no-interaction --no-progress --optimize-autoloader"
 
 services:
-  - memcached
+    - memcached
```
Comment (samdark) [inline .travis.yml]: Wasn't memcached removed?

File: phpunit.xml.dist [hunk 2]
```diff
@@ -23,6 +23,7 @@
             <exclude>
                 <directory>./tests</directory>
                 <directory>./vendor</directory>
+                <file>./src/NullCache.php</file>
```
Comment (samdark) [inline phpunit.xml.dist]: Why?

Comment (alexkart) [reply phpunit.xml.dist]: Do we need to add tests for it?

File: .travis.yml (hunk 1 above)
Comment (alexkart) [reply .travis.yml]: Yes

File: phpunit.xml.dist (hunk 2 above)
Comment (samdark) [reply phpunit.xml.dist]: Why not? https://github.com/yiisoft/cache/blob/7b6be46688f83e272972f61e00ac22a1d6dd0332/tests/NullCacheTest.php

Comment (alexkart) [reply phpunit.xml.dist]: I didn't notice it.

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**